examples/marko-cli/src/stories/addon-knobs.stories.js

```diff
@@ -5,16 +5,14 @@ export default {
   title: 'Addons|Knobs/Hello',
   decorators: [withKnobs],
   parameters: {
-    component: Hello,
+    template: Hello,
```

Comment:
component param is for addon-docs

Reply:
Thanks for the pointer! Could you share with me the docs for that config option? I’m new to using story book and want to make sure that Marko is accommodating as much of the ecosystem as possible 🙂.

Reply:
It hasn't made it into the main docs yet (coming soon!), but here's the best resource for now. https://docs.google.com/document/d/1un6YX7xDKEKl5-MVb-egnOYN8dynb5Hf7mq0hipk8JE/edit

Reply:
Excellent, thanks. Perhaps it would make sense to use `component` instead of `template` as a property for the story fns as well just to be consistent. I’ll update the PR shortly.